DESCRIPTION

```diff
@@ -1,16 +1,19 @@
 Package: rtables
 Title: Reporting Tables
-Version: 0.6.11.9019
+Version: 0.6.12
 Date: 2025-04-08
```

Comment:
```suggestion
Date: 2025-04-10
```

Reply:
if there is a recent commit, yes. but I think there was a merge from main, and reverted the date